### PR TITLE
Make maturing monster keep their names

### PR DESCRIPTION
**Pull request by Grummel7**
See https://www.freeorion.org/forum/viewtopic.php?f=28&t=12370&sid=34368ea791f2c5c53c72e2ae9b35983b

Merged Krill Swarms get a new name, or should they keep one of the old ones? I guess nobody really cares for the names of Krill Swarms...

### CODE REVIEW

File: default/scripting/ship_parts/General/SP_CLOUD.focs.txt [hunk 1]
```diff
@@ -25,23 +25,23 @@ Part
...
+                CreateShip designname = "SM_DIM" empire = Source.Owner name = Source.Name
                 Destroy
             ]
 
         EffectsGroup
             scope = Source
             activation = Number low = 1 condition = And [Capital OwnerHasTech name = "SPY_STEALTH_4"]
             effects = [
-                CreateShip designname = "SM_VOID" empire = Source.Owner
+                CreateShip designname = "SM_VOID" empire = Source. name = Source.Name
```
Comment (agrrr3) [inline default/scripting/ship_parts/General/SP_CLOUD.focs.txt]: Source.  ->  Source.Owner

Comment (Grummel7) [reply default/scripting/ship_parts/General/SP_CLOUD.focs.txt]: Argh, yes, of course...

**Comment by geoffthemedio** [comment]
> Merged Krill Swarms get a new name, or should they keep one of the old ones?

It may be possible to combine them with the + operator

**Comment by Grummel7** [comment]
> It may be possible to combine them with the + operator

You mean concatenating the name? They can merge up to 3 times, that can produce very long names.
Also I have no idea how to get the names of the other swarms.

**Pull request merged**